patches/tModLoader/Terraria/ModLoader/TileLoader.cs

```diff
@@ -655,7 +657,7 @@ public static void SetDrawPositions(int i, int j, ref int width, ref int offsetY
 			GetTile(tile.type).SetDrawPositions(i, j, ref width, ref offsetY, ref height, ref tileFrameX, ref tileFrameY);
 		}
 	}
-	
+
```

Comment:
Please avoid submitting formatting/whitespace changes in PRs in the future. No need to change for now, because we may as well clean that up, but it does pollute the review and risk merge conflicts.

Reply:
Is there a way I can make sure the patches don't add that in? I didn't make those changes myself so I'm not really sure where they came from unless I have my formatting setup incorrectly. I'll definitely review my preferences in VS.

Reply:
Just make sure auto-format doesn't apply to the entire file